test_app/Rakefile

```diff
@@ -0,0 +1,6 @@
+# Add your own tasks in files placed in lib/tasks ending in .rake,
```

Comment:
What is this file about?

Reply:
Rake is a build language based on Ruby. In Rails, It is used for common administration or repetitive tasks, especially sophisticated ones that build off of each other. 

The 'Rakefile' and .rake files are used by the Rake to build up a list of tasks. We can add rake tasks by creating .rake files in the lib/tasks folder of our Rails project.

Reply:
Define `build language`

Reply:
Build Language is an extensible build automation DSL for declaratively defining builds. A Domain-Specific Language (DSL) is a computer language that is designed to solve a specific type of problem rather than being a general-purpose language that can solve any type of software problem such as HTML specific for web development.  Rake is referred to as a "build tool" because it includes libraries that enable standard build/deploy such as file operations (creating, removing, renaming, and moving files), publishing sites through FTP/SSH, and running simple tests.